### DISCUSSION TITLE
reenable news schedulers - no new messages from 20.07

### DISCUSSION THREAD

**Discussion by blka** [xueyuanl/daily-hackernews#1414] [General]
hi, @Thommas2316 / @xueyuanl  can you guys enable newsletter again please? - se the `action` tab:
"This scheduled workflow is disabled because there hasn't been activity in this repository for at least 60 days. Enable this workflow to resume scheduled runs."

**Comment by Thommas2316** [comment]
Hi @blka,

I sent over a PR for @xueyuanl to merge, so it should start wokring after it is merged :)

Have a nice day :)

**Comment by ariasalvaro** [comment]
The feed is again stopped, since April 20. Can it be reenabled, @xueyuanl? Thank you!
